### PR TITLE
Force renaming parameters for experiments

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Force renaming parameters for short names in `ExperimentFunction` so that it is printable in the experiments plots.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/ml/mlfunctionlib.py [hunk 1]
```diff
@@ -163,16 +151,10 @@ def __init__(
...
-            # For the test we just switch noise_free to True.
-            self.evaluation_function = partial(self._ml_parametrization, criterion="mse",  # type: ignore
-                                               min_samples_split=0.00001,
-                                               regressor="decision_tree", noise_free=not overfitter,
-                                               alpha=1.0, learning_rate="no",
-                                               activation="no", solver="no")
+            params = dict(noise_free=False,
+                          alpha=1.0, learning_rate="no", regressor="decision_tree",
+                          activation="no", solver="no")
+            evalparams = dict(params, criterion="mse", min_samples_split=0.00001)
```
Comment (jrapin) [inline nevergrad/functions/ml/mlfunctionlib.py:157]: @teytaud this is probably not doing at all what you want:
You would like the evaluation to be mse only? But the parametrization may override these parameters, it's not the other way round, have you checked it or must it be checked?

**Review by jrapin — COMMENTED** [review]
@teytaud this is to make sure that all parametrization names are set to something short enough to be printable. I'd like to get rid of the plot names being random strings.

**Comment by teytaud** [comment]
Thanks for this idea.
This prevents lsgo from running.

File: nevergrad/functions/ml/mlfunctionlib.py (hunk 1 above)
Comment (teytaud) [reply nevergrad/functions/ml/mlfunctionlib.py:157]: Ah thanks for pointing out this issue.
I've seen your other PR, I'll play with it, thanks for this as well.

**Pull request merged**